data/en/isbinary.json

```diff
@@ -17,6 +17,15 @@
 	},
 	"links": [
 
-	]
+	],
 
+	"examples": [
+		{
+			"title":"isBinary example",
+			"description":"returns true if given value is binary data, returns false if otherwise",
+			"code":"isBinary( toBase64(1) )",
+			"result":true,
```

Comment:
[trycf](https://trycf.com/) and [travisCI](https://travis-ci.org/foundeo/cfdocs/jobs/291892460#L562) say, this is evaluated to `false` instead of `true`.